Lib/fontbakery/profiles/shared_conditions.py

```diff
@@ -157,16 +157,45 @@ def glyph_metrics_stats(ttFont):
     """Returns a dict containing whether the font seems_monospaced,
     what's the maximum glyph width and what's the most common width.
 
-    For a font to be considered monospaced, at least 80% of
-    the ascii glyphs must have the same width."""
+    For a font to be considered monospaced, if at least 80% of ASCII
+    characters have glyphs, then at least 80% of must have the same width,
+    otherwise all glyphs of printable characters must have one of two widths
+    or be zero-width.
+    """
     glyph_metrics = ttFont['hmtx'].metrics
-    ascii_glyph_names = [ttFont.getBestCmap()[c] for c in range(32, 128)
+    ascii_glyph_names = [ttFont.getBestCmap()[c] for c in range(32, 127)
```

Comment:
[007F](https://r12a.github.io/uniview/?char=007F) is a control character. We don't include 0000-0031 because they are control characters, 007F shouldn't be included either.

Reply:
we should add a comment about that in the source so that other people are aware of this reasoning in the future